app/models/tagged_news.rb

```diff
@@ -1,12 +1,22 @@
-class TaggedNews
+class TaggedNews < SimpleDelegator
   def self.all(article)
     news_items = Mas::Cms::News.all(
       params: {
         document_type: [Mas::Cms::News::PAGE_TYPE],
         tag: Array(article.tags)
       }
-    )
+    ).reject { |news_item| news_item.slug == article.slug }
 
-    news_items.reject { |news_item| news_item.slug == article.slug }
+    news_items.map do |news_item|
+      new(news_item)
+    end.sort_by(&:published_date).reverse
+  end
```

Comment:
same comment as on lines 20-22 of news_controller

Reply:
:recycle: